### DISCUSSION TITLE
DNS Check: Allow CNAME to pass

### DISCUSSION THREAD

**Discussion by genox** [coollabsio/coolify#1912] [Feature Requests]
Hi,

I am using CNAMEs to forward traffic to coolify, not A records. I think this is why my DNS checks fail with:

```
Validating DNS (https://somedomain.com) failed.
Make sure you have added the DNS records correctly.

Check this [documentation](https://coolify.io/docs/knowledge-base/dns-configuration) for further help.
```

It would be great if Coolify would be able to handle that.

Cheers and thank you.

**Comment by jsmits** [comment]
For me that works fine. I use CNAMEs to forward traffic to my various services on my Coolify server. Using Cloudflare for DNS, but that shouldn’t really matter, I guess. I do use sub-domains though, so that might be different from what you’re doing. And I point those CNAMEs to an A record that points to my Coolify server IP.

**Comment by genox** [comment]
Maybe that has been fixed meanwhile. I will give it a try.

In order to troubleshoot issues like that an addition to the docs of a) what exactly is being checked and b) where to find the checks error output would be very helpful.
